src/FilterResource.php

```diff
@@ -52,7 +52,7 @@ public function append($append)
      */
     public function bool()
     {
-        return $this->addRule(new FilterRule\Bool);
+        return $this->addRule(new FilterRule\Boolean);
```

Comment:
I know technically this resolves the reserved words incompatibility, but why not use the same naming scheme as all the others: CastBool?